src/fuzz.rs

```diff
@@ -96,7 +96,7 @@ fn process_and_sort(s: &str, force_ascii: bool, full_process: bool) -> String {
         s.to_string()
     };
     let mut ts_split: Vec<_> = ts.split_whitespace().collect();
-    ts_split.sort();
+    ts_split.sort_unstable();
```

Comment:
Are there any negative side-effects from no longer guaranteeing preserved ordering?

Reply:
Not that I know of. Because we are sorting `&str`, it is not possible to detect any difference between a stable and unstable sort. We don't care when we sort `["foo", "bar", "foo"]` which `"foo"` ends up last, because we cannot detect which `"foo"` ends up last.